### ISSUE TITLE
Problem loading TIA (.emi, .ser) files

### ISSUE THREAD

**Issue by viviannh81** [hyperspy/hyperspy#2238]
I am using v1.4.2 installed by Anaconda on windows.
When uploading a .ser or .emi files, it produces the following error, while using v1.4.1 installed by the bundle it works for .ser
```python
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-15-88f6df8068f9> in <module>
----> 1 s = hs.load("FeP_Lame1_OSIRIS_7757_3.ser")
      2 s

~\Anaconda3\lib\site-packages\hyperspy\io.py in load(filenames, signal_type, stack, stack_axis, new_axis_name, lazy, convert_units, **kwds)
    272             objects = [load_single_file(filename, lazy=lazy,
    273                                         **kwds)
--> 274                        for filename in filenames]
    275 
    276         if len(objects) == 1:

~\Anaconda3\lib\site-packages\hyperspy\io.py in <listcomp>(.0)
    272             objects = [load_single_file(filename, lazy=lazy,
    273                                         **kwds)
--> 274                        for filename in filenames]
    275 
    276         if len(objects) == 1:

~\Anaconda3\lib\site-packages\hyperspy\io.py in load_single_file(filename, **kwds)
    309     else:
    310         reader = io_plugins[i]
--> 311         return load_with_reader(filename=filename, reader=reader, **kwds)
    312 
    313 

~\Anaconda3\lib\site-packages\hyperspy\io.py in load_with_reader(filename, reader, signal_type, convert_units, **kwds)
    316     lazy = kwds.get('lazy', False)
    317     file_data_list = reader.file_reader(filename,
--> 318                                         **kwds)
    319     objects = []
    320 

~\Anaconda3\lib\site-packages\hyperspy\io_plugins\fei.py in file_reader(filename, *args, **kwds)
    286     ext = os.path.splitext(filename)[1][1:]
    287     if ext in ser_extensions:
--> 288         return [ser_reader(filename, *args, **kwds), ]
    289     elif ext in emi_extensions:
    290         return emi_reader(filename, *args, **kwds)

~\Anaconda3\lib\site-packages\hyperspy\io_plugins\fei.py in ser_reader(filename, objects, *args, **kwds)
    473 
    474     """
--> 475     header, data = load_ser_file(filename)
    476     record_by = guess_record_by(header['DataTypeID'])
    477     ndim = int(header['NumberDimensions'])

~\Anaconda3\lib\site-packages\hyperspy\io_plugins\fei.py in load_ser_file(filename)
    333                                   dtype=np.dtype(
    334                                       data_dtype_list + tag_dtype_list),
--> 335                                   count=1)
    336             f.seek(offset)
    337         _logger.info("Data info:")

ValueError: setting an array element with a sequence.
```

The attachment is one of the related files
[FeP_Lame1_7768.zip](https://github.com/hyperspy/hyperspy/files/3450844/FeP_Lame1_7768.zip)

**Comment by ericpre** [comment]
I can reproduce the error with hyperspy 1.4.2 and this is fixed in the more recent releases (1.5 and 1.5.1), try to update hyperspy.

```
conda update hypersy
```

depending on how your conda environment is set up, most likely you will need to add `-c conda-forge`

**Comment by viviannh81** [comment]
I tried it out, in v1.5.1 it's fixed!
